finesse/gui/measure_script/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Measure scripts represent a sequence of mirror positions for a recording.
+
+This module contains code for saving and loading mirror scripts, as well as creating and
```

Comment:
mirror scripts -> measure scripts?

Reply:
Good spot!

Reply:
Well, actually itbis both, measure and mirror scripts as they set the position pf the mirror

Reply:
True. I did toy with calling them "mirror scripts" initially, but eventually I settled on calling them "measure scripts" everywhere to avoid confusion.